Makefile

```diff
@@ -44,7 +44,10 @@ python_test: python3 tox
 go_build: git mockgen packr2
 	make webapp_node_modules_prod
 	go generate ./...
+	# Check all packages without producing any output.
 	go build ./...
+	# Build the webapp.
+	go build ./webapp/web
```

Comment:
why is this being run separately? it wasn't built prior to this change?

Reply:
No, when given multiple packages (e.g. `./...`), `go build` does not store the output, so we'd need to have a separate invocation. This is built for local dev (as well as large/webdriver tests). Essentially this binary is the entrypoint to our server both on prod and locally now.